### ISSUE TITLE
[MINIPLUS] [PORTS] Some ports aren't working on the update 0.2

### ISSUE THREAD

**Issue by efylan** [game-de-it/plumOS-GKD#2]
Hello, thank you for keeping the updates available for the GKD mini PLUS (peak hardware design!!)

Some ports are not running, (they are working on the 20230411 version though) I tried reinstalling them and patching them again.

The ones I have tested are

-Undertale

-Delta Rune

-Risk of Rain

I will keep testing and add them here.

**Comment by game-de-it** [comment]
Thanks for reporting.

plumOS v0.1 and v0.2 are based on different versions of stockOS.
Most ports games work on v0.1, but conversely, many ports games do not work on v0.2.

We have confirmed that the game crashes when playing audio, but have not found a solution.
We intend to document this symptom as a known issue in v0.3.

**Comment by efylan** [comment]
I think I found out ports that are not working are .apk based games that depend on gmloader giving out this error:

`Failed to restart oga_events.service: Unit oga_events.service not found.
`

**Comment by game-de-it** [comment]
Yes, since `oga_events.service` itself does not exist in GKD-StockOS, the error should occur in v0.1 as well.
However, since no ports bugs have been reported in v0.1, the existence of `oga_events.service` is not directly related to the issue.

**Comment by efylan** [comment]
> Thanks for reporting.
> 
> plumOS v0.1 and v0.2 are based on different versions of stockOS. Most ports games work on v0.1, but conversely, many ports games do not work on v0.2.
> 
> We have confirmed that the game crashes when playing audio, but have not found a solution. We intend to document this symptom as a known issue in v0.3.

I think I found out ports that are not working are .apk based games that depend on **gmloader** giving out this error:

`Failed to restart oga_events.service: Unit oga_events.service not found.
`

I'm not sure if tt's missing [**this**](https://github.com/christianhaitian/arkos/blob/4da40191f6320f1eca38a0144f627b043d1cc93c/scripts/oga_events.py) script, which is part of the arkos project to reset inputs, I couldn't find it anywhere on the 2023 firmware, so maybe if it's not about this file being missing, maybe it has to do with systemd running in a mode where not finding a service results in an exception/wrong exit code

This [search](https://github.com/search?q=owner%3Achristianhaitian%20oga_events&type=code) returned some code that gave me a few ideas, but as I said, I could't find the oga_events.service anywhere...

**Comment by game-de-it** [comment]
Oops, I forgot to check in the first place, but does the game you are trying to run run on plumOS v0.1?

I understand that many ports games cannot be run on v0.2, but various users have reported that they work fine on v0.1.

**Comment by efylan** [comment]
> Oops, I forgot to check in the first place, but does the game you are trying to run run on plumOS v0.1?
> 
> I understand that many ports games cannot be run on v0.2, but various users have reported that they work fine on v0.1.

Aww unfortunately running the v0.1 image on the miniplus results on a system failure screen, I tried replacing some files but still couldn't get it running. For now I'll stick to the 2023 firmware for ports and the new one for emulation.

**Comment by game-de-it** [comment]
Don't worry about it.
If the SD card is broken, various symptoms will occur, but I don't know all of them, so this experience was very educational.(^^)
